library/src/main/kotlin/com/vinted/actioncable/client/kotlin/Connection.kt

```diff
@@ -35,6 +36,7 @@ class Connection constructor(
             var query: Map<String, String>? = null,
             var headers: Map<String, String>? = null,
             var reconnection: Boolean = false,
+            var infinityReconnection: Boolean = false,
```

Comment:
`infinityReconnection` competes with `reconnectionMaxAttempts`. I see two potential solutions to avoid ambiguity:
- Create constant `const val INFINITY_RECONNECTION = Integer.MAX_VALUE`. There is no chance of exhausting 2 mlrd. repeats in a lifetime :) 
- Create `ReconnectionPolicy`, which will be more expressive and gives more flexibility of handling connections. 

The first solution is ABI compatible.

Reply:
How i can delete readme files from my fork?

Reply:
it my first fork in my life, sorry (

Reply:
What the ReconnectionPolicy?

Reply:
> it my first fork in my life, sorry (

Don't worry, no problem. I hope we could teach you a thing or two. :)

Reply:
> What the ReconnectionPolicy?

That class does not exist. It is just an idea, to create policy class which could have implementation for various reconnection behaviors. For example, one variant could be infinity connections. However, I am not recommending doing that, because of simple `Integer.MAX_VALUE` will solve the problem and it is binary and source backward compatible.

Reply:
> How i can delete readme files from my fork?

I suppose the easiest way is to do that is force update your master. Ir you could revert. However, this PR will stick to your master branch, so any code you make there will effect this PR. My recommendation next time to create new branches. Sometimes it makes thinks easier.

Reply:
> > What the ReconnectionPolicy?
> 
> That class does not exist. It is just an idea, to create policy class which could have implementation for various reconnection behaviors. For example, one variant could be infinity connections. However, I am not recommending doing that, because of simple `Integer.MAX_VALUE` will solve the problem and it is binary and source backward compatible.

Thankyou) I will fix this moment)

Reply:
> > How i can delete readme files from my fork?
> 
> I suppose the easiest way is to do that is force update your master. Ir you could revert. However, this PR will stick to your master branch, so any code you make there will effect this PR. My recommendation next time to create new branches. Sometimes it makes thinks easier.

Ok) I can try